Remove gfpgan_dir

blessedcoolant opened the pull request:
gfpgan_dir is no longer needed because we are now installing gfpgan directly to the env. The user can just pass the path to the model file directly now using `--gfpgan_model_path` if they want to use a custom path for the checkpoint.

blessedcoolant commented:
@lstein I want another review on this before I merge. Thank you.

lstein reviewed (commented):
I've made changes to how and where the gfpgan model files are handled in the current release candidate, and would like to hold off on this until those changes are merged into main and development. Shouldn't be more than another day, pending install testing.

blessedcoolant commented:
> I've made changes to how and where the gfpgan model files are handled in the current release candidate, and would like to hold off on this until those changes are merged into main and development. Shouldn't be more than another day, pending install testing.

are they being sent to models/gfpgan? I can change the paths to that

lstein commented:
> > I've made changes to how and where the gfpgan model files are handled in the current release candidate, and would like to hold off on this until those changes are merged into main and development. Shouldn't be more than another day, pending install testing.
> 
> are they being sent to `models/gfpgan/weights`? I can change the paths to that

Yes, you should. Although I must already have done this in the release candidate because it's working :-). `models` currently looks like this:
```
models
├── clipseg
│   └── clipseg_weights
│       ├── rd16-uni.pth
│       ├── rd64-uni-refined.pth
│       └── rd64-uni.pth
├── gfpgan
│   ├── GFPGANv1.4.pth
│   └── weights
│       ├── detection_Resnet50_Final.pth
│       └── parsing_parsenet.pth
└── ldm
    └── stable-diffusion-v1
        ├── kl-f8-anime.ckpt
        ├── model-epoch09-float32.ckpt
        ├── model-epoch09-float32.sha256
        ├── place-ckpt-files-here.txt
        ├── sd-v1-5-inpainting.ckpt
        ├── sd-v1-5-inpainting.sha256
        ├── v1-5-pruned-emaonly.ckpt
        ├── v1-5-pruned-emaonly.sha256
        ├── vae-ft-mse-840000-ema-pruned.ckpt
        ├── wd-1-3-penultimate-ucg-cont.ckpt
        └── wd-1-3-penultimate-ucg-cont.sha256
```

blessedcoolant commented:
@lstein Done. Updated to the new paths according to the folder hierarchy you posted. Think I changed it across the entire app. Just do a double check and it should be good to go.
